src/walk/index.js

```diff
@@ -139,11 +139,22 @@ export function findNodeBefore(node, pos, test, base, state) {
   return max
 }
 
+// Fallback to an Object.create polyfill for older environments.
+const create = Object.create ? Object.create : (() => {
+  let Temp = function() {}
+  return function(prototype) {
+    Temp.prototype = prototype
+    let result = new Temp()
+    Temp.prototype = null
```

Comment:
This creates extra `.prototype` property on object that didn't have it initially.